### DISCUSSION TITLE
Search bugs

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#750] [0. General bugs and questions]
The new search is buggy. Noticing the following issues:

-Results are repeating.
-Other version results are showing in another version when choosing the "only in..." option.
-With the "All the content" option, the same results are not showing from all versions.

Problem exhibited in the screen recording below. If there is a possibility, model this back on the old search.

https://github.com/user-attachments/assets/4dda579c-a285-4ce7-8690-1fc7dc3e0cc4

**Comment by scazan** [comment]
> The new search is buggy. Noticing the following issues:
> 
> -Results are repeating. -Other version results are showing in another version when choosing the "only in..." option. -With the "All the content" option, the same results are not showing from all versions.
> 
> Problem exhibited in the screen recording below. If there is a possibility, model this back on the old search.
> 
>  search.bugs.mov

Hi @ahcsco , we'll check into this right away.

**Reply by ahcsco** [reply]
> > The new search is buggy. Noticing the following issues:
> > -Results are repeating. -Other version results are showing in another version when choosing the "only in..." option. -With the "All the content" option, the same results are not showing from all versions.
> > Problem exhibited in the screen recording below. If there is a possibility, model this back on the old search.
> > search.bugs.mov
> 
> Hi @ahcsco , we'll check into this right away.

Thanks for your quick response. Awaiting...

**Reply by scazan** [reply]
Just to give you an update. We found the issue and the fix is coming soon. Looks like it was related to content that is duplicated from one space to another and this bug surfaces when that happens. Thanks for reporting it!

**Reply by ahcsco** [reply]
Hi, when will the fix be issued for this?

**Reply by scazan** [reply]
Hi! Sorry about delays on it. The fix is currently being reviewed for deployment so I expect it will be shipped very soon.

**Reply by ahcsco** [reply]
Thank you.

**Reply by scazan** [reply]
Hi @ahcsco , this should be resolved as of the weekend now regarding the duplicates showing in multiple variants. I noticed another small issue we are working to resolve now as well that is unrelated regarding switching the filter back to a single variant but that issue is a bit different.

**Reply by ahcsco** [reply]
Hi and thanks. I believe i can notice a change in search behavior now. However, at least issue # 2 (-Other version results are showing in another version when choosing the "only in..." option.) is still observed. 
in the attached video, i see results from 6.3 and 6.2 on 6.4.

https://github.com/user-attachments/assets/870fe4ba-f3bf-4dd4-95b6-313bd30a37b4
